HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -275,6 +275,10 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
 
     float d0 = 0.0;
     float dz = 0.0;
+    float trackpt = -9999.0;
+    float tracketa = -9999.0;
+    float trackphi = -9999.0;
+    float trackchi2overndf = -9999.0;
```

Comment:
```suggestion
    float trackpt = 0.;
    float tracketa = 0.;
    float trackphi = 0.;
    float trackchi2overndf = 0.;
```
Please use `0.` to be consistent with what is done for `d0` and `dz`.

Please take this chance to squash the whole PR into 1 commit.


(As far as I understand, the value, `0.` or `-9999.0`, does not really matter as it is never present in the outputs. When those variables are not updated with meaningful values (photon case, `charge == -999`), they are not used to fill the output products.)